src/components/molecules/CastCard/index.tsx

```diff
@@ -0,0 +1,196 @@
+import React, { memo } from "react";
+import { styled } from "@pigment-css/react";
+
+import Avatar from "../../atoms/Avatar";
+import { useLinkifyBio } from "../../organisms/NeynarProfileCard/hooks/useLinkifyBio";
+import Box, { HBox, VBox } from "../../atoms/Box";
+import { WarpcastPowerBadge } from "../../atoms/icons/WarpcastPowerBadge";
+import { useRenderEmbeds } from "../../organisms/NeynarCastCard/hooks/useRenderEmbeds";
+import Reactions from "./Reactions";
+
+const StyledCastCard = styled.div(({ theme }) => ({
+  display: "flex",
+  flexDirection: "column",
+  width: "100%",
+  maxWidth: "608px",
+  borderWidth: "1px",
+  borderStyle: "solid",
+  borderColor: theme.vars.palette.border,
+  borderRadius: "15px",
+  padding: "30px",
+  color: theme.vars.palette.text,
+  fontFamily: theme.typography.fonts.base,
+  fontSize: theme.typography.fontSizes.medium,
+  backgroundColor: theme.vars.palette.background,
+}));
+
+const StyledLink = styled.a(({ theme }) => ({
+  textDecoration: "none",
+  color: theme.vars.palette.textMuted,
+}));
+
+const Main = styled.div(() => ({
```

Comment:
for sure -- i still might need to wrap it or add some extra styling though because the space-between logic for this `Main` const separates these icons below
<img width="580" alt="image" src="https://github.com/neynarxyz/react/assets/12853808/e0514c01-de92-4e52-93e6-28a8f20f062c">